src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java21/ExpectedStringTemplateBasicWithTabs.txt

```diff
@@ -222,22 +210,18 @@ COMPILATION_UNIT -> COMPILATION_UNIT [2:0]
         |   |           |--DOT -> . [31:21]
         |   |           |   |--IDENT -> STR [31:18]
         |   |           |   `--STRING_TEMPLATE_BEGIN -> " [31:23]
-        |   |           |       |--STRING_TEMPLATE_CONTENT ->  [31:24]
         |   |           |       |--EMBEDDED_EXPRESSION_BEGIN -> \{ [31:24]
         |   |           |       |--EMBEDDED_EXPRESSION -> EMBEDDED_EXPRESSION [31:27]
         |   |           |       |   `--IDENT -> x [31:27]
         |   |           |       |--EMBEDDED_EXPRESSION_END -> } [31:29]
-        |   |           |       |--STRING_TEMPLATE_CONTENT ->  [31:30]
         |   |           |       `--STRING_TEMPLATE_END -> " [31:30]
         |   |           `--DOT -> . [31:38]
         |   |               |--IDENT -> STR [31:35]
         |   |               `--STRING_TEMPLATE_BEGIN -> " [31:40]
-        |   |                   |--STRING_TEMPLATE_CONTENT ->  [31:41]
```

Comment:
I don't see any changes to `TokenTypes` and our ANTLR displays. If we don't have a UT requiring these to be kept in synch, these kind of changes will break `TokenTypes`.

Also, did we remove all usages of `STRING_TEMPLATE_CONTENT `? Should this token now be undocumented?